test/functional/wallet_send.py

```diff
@@ -133,6 +139,11 @@ def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
             assert not "txid" in res
             assert "psbt" in res
 
+        if include_unsafe:
+            from_balance = from_wallet.getbalances()["mine"]["untrusted_pending"]
+        else:
+            from_balance = from_wallet.getbalances()["mine"]["trusted"]
```

Comment:
same here
```python
        from_balance = from_wallet.getbalances()["mine"]["trusted"]
        if include_unsafe:
            from_balance += from_wallet.getbalances()["mine"]["untrusted_pending"]
```

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/21359/commits/0709545365d045b24563b4683881f65a2b34afdd